src/wp-includes/block-bindings/sources/post-meta.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Add the post_meta source to the Block Bindings API.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function post_meta_source_callback( $source_attrs ) {
+	if ( ! isset( $source_attrs['key'] ) ) {
+		return null;
+	}
+
+	// Use the postId attribute if available
```

Comment:
Missing end period.

Reply:
I opened https://github.com/WordPress/wordpress-develop/pull/6016 to address the feedback.